webpack.dev.config.ts

```diff
@@ -62,6 +62,13 @@ module.exports = {
     {
       test: /\.html$/,
       loader: 'underscore-template-loader'
+    },
+    {
+      test: /\.css$/,
+      include: [
+        path.resolve(__dirname, 'extensions'),
+      ],
+      use: ['style-loader', 'css-loader']
```

Comment:
Do we need `css-loader`? Isn't `style-loader` enough?

Reply:
Yes we need `css-loader`
`style-loader` depends on `css-loader`